packages/block-library/src/file/edit.js

```diff
@@ -207,7 +207,7 @@ class FileEdit extends Component {
 									tagName="div" // must be block-level or else cursor disappears
 									className={ 'wp-block-file__button' }
 									value={ downloadButtonText }
-									formattingControls={ [] } // disable controls
+									disallowFormats // disable controls
```

Comment:
Why prevent the user from having formatting in the button? `<a>`, `<b>`, `<i>`, etc. are allowed inside a `<button>`.

Reply:
I think it has a wrapping anchor element?

Reply:
Now possible. `a` cannot be used as it is an interactive element.